recipe/meta.yaml

```diff
@@ -63,24 +63,24 @@ outputs:
         - cudatoolkit      # [(cuda_compiler_version or "").startswith("11")]
         - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
         - nccl             # [cuda_compiler_version != "None"]
-        - librmm ={{ rapids_version }}  # [linux]
+        - librmm >={{ rapids_version }}  # [linux and cuda_compiler_version != "None"]
       run:
         - llvm-openmp  # [osx]
-        - librmm ={{ rapids_version }}  # [linux]
+        - librmm >={{ rapids_version }}  # [linux and cuda_compiler_version != "None"]
         - __cuda  # [cuda_compiler_version != "None"]
 
   - name: _py-xgboost-mutex
     version: 2.0
     build:
-      string: cpu_0  # [cuda_compiler_version == "None"]
-      string: gpu_0  # [cuda_compiler_version != "None"]
+      string: rapidsai_cpu_0  # [cuda_compiler_version == "None"]
+      string: rapidsai_gpu_0  # [cuda_compiler_version != "None"]
```

Comment:
Since there is nothing RAPIDS specific about the mutex packages, would it make sense to leave these as-is (without the `rapids_` prefix in the `build/string`?